datatracer/data.py

```diff
@@ -46,10 +47,11 @@ def load_datasets(datasets_path):
         dict:
             Dict of (metadata, tables) tubles, one for each dataset.
     """
-    return {
-        dataset: load_dataset(os.path.join(datasets_path, dataset))
-        for dataset in os.listdir(datasets_path)
-    }
+    datasets = {}
+    for path_to_dataset in glob(os.path.join(datasets_path, "*/")):
```

Comment:
Why this change?

Reply:
This was specifically to avoid `.DS_Store` files, but it seems generally reasonable to limit it to folders. It might even make sense to glob for `*/metadata.json` to only select folders that resemble datasets.